ark/devnet/network.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Why would this be here? This is already included in the crypto package and useless duplication.

Reply:
This would be the canonical place, so any project could load the configuration. Some people, who don't know `crypto` wouldn't know where to find this info.